src/phoenix/server/api/routers/v1/experiment_evaluations.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime
+
+from starlette.requests import Request
+from starlette.responses import JSONResponse, Response
+from starlette.status import HTTP_404_NOT_FOUND
+from strawberry.relay import GlobalID
+
+from phoenix.db import models
+from phoenix.server.api.types.node import from_global_id_with_expected_type
+
+
+async def create_experiment_evaluation(request: Request) -> Response:
+    experiment_run_globalid = GlobalID.from_id(request.path_params["run_id"])
+    try:
+        experiment_run_id = from_global_id_with_expected_type(
+            experiment_run_globalid, "ExperimentRun"
+        )
+    except ValueError:
+        return Response(
+            content=f"ExperimentRun with ID {experiment_run_globalid} does not exist",
+            status_code=HTTP_404_NOT_FOUND,
+        )
+
+    payload = await request.json()
+    name = payload.get("name")
+    label = payload.get("label")
+    score = payload.get("score")
+    explanation = payload.get("explanation")
+    error = payload.get("error")
+    metadata = payload.get("metadata", {})
+    start_time = payload.get("start_time")
+    end_time = payload.get("end_time")
```

Comment:
I think we need to be a little more defensive here when marshalling the payload for required types, because if I recall SQLite will insert anything into the DB, even nulls in required columns. I actually feel like we should probably we doing type checking.

Reply:
I think we should avoid that sort of thing for now, validation code belongs in our object serializers once we build them IMO